BaseLib/ConfigTree.h

```diff
@@ -338,24 +338,6 @@ class ConfigTree final
      */
     //!\{
 
-    /*! Get parameter \c param from the configuration tree.
-     *
-     * \return the subtree representing the requested parameter
-     *
-     * \pre \c param must not have been read before from this ConfigTree.
-     */
-    ConfigTree
-    getConfParam(std::string const& param) const;
```

Comment:
I deleted because nobody use these methods now. anyway if you have a reason for it, I keep them.